docs/components/modeler/web-modeler/milestones.md

```diff
@@ -1,50 +1,158 @@
+- Only differences that affect the execution of the BPMN process are highlighted.
+- The **Changes** panel shows the details of each change, including the type and identifier. Select a change to highlight it.
+
+:::note
+
+You can only use the **Code view** to compare changes in a DMN diagram. The **Visual view** only shows a view of the milestone.
+
+:::
+
+### Compare milestones in code view
 
-Hover over a milestone, click on the three vertical dots, and expand for more options.
-![milestones restore](img/milestones/web-modeler-milestone-restore.png)
-![milestones restore](img/milestones/web-modeler-milestone-restore-complete.png)
+To view BPMN and DMN diagram changes as code in an XML diff layout, select the **Code view** tab.
 
-### Comparing milestones
+![milestones diffing](img/milestones/web-modeler-milestone-code-diffing.png)
 
-Milestones can be compared visually. By enabling the diffing feature, the currently selected milestone is compared to its predecessor.
+- The XML for the previous milestone is shown on the left, with the currently selected milestone shown on the right.
+- Differences between the milestones are highlighted in the XML. For example, if an element was added, this change is highlighted in green.
+- Only differences that affect the execution of the BPMN process are highlighted.
 
-The differences that are highlighted are only those that affect the execution of the BPMN process. Pure visual changes like position changes are not highlighted.
-![milestones diffing](img/milestones/web-modeler-milestone-diffing.png)
+### Compare process application milestones
+
+You can also use the **Code view** tab to view and compare the [versioned milestones](process-applications.md#versioning) of individual files in a process application.
+
+When you view the milestone history for any file in a process application, you can also navigate between the files to view and compare their milestone history.
+
+![milestones restore](img/milestones/web-modeler-milestone-compare-process-application-files-highlight.png)
+
+:::note
+
+You cannot compare two process application versions using the **Code view**, you can only view and compare milestones for each individual process application file.
+
+:::
+
+## Restore a milestone
+
+You can restore a milestone to revert to a previous snapshot of your diagram.
+
+1. In the **Milestones** panel, hover over the milestone you want to restore.
+1. Select the three vertical dots to open the actions menu.
+1. Select **Restore as latest**.
+
+![milestones restore](img/milestones/web-modeler-milestone-restore-highlight.png)
+
+The diagram reverts to the restored milestone. A new change milestone is created with "(restored)" appended to the name.
+
+![milestones restore](img/milestones/web-modeler-milestone-restore-complete-highlight.png)
+
+## Copy a diagram milestone
+
+You can make a copy of a diagram for a specific milestone.
```

Comment:
🔧 
```suggestion
You can create a new diagram from a specific milestone.
```

Reply:
Edited the Copy section to reflect this change in terminology.